docs/platforms/javascript/guides/nuxt/install/limited-server-tracing.mdx

```diff
@@ -1,20 +1,21 @@
 ---
-title: Top-level import
+title: Limited Server Tracing
```

Comment:
Not sure, do we need to add a redirect for this renamed page? 🤔 Not sure how many links to the /top-level-import path are reasonably out there...?

Reply:
It was online for quite some time so maybe not a bad idea